examples/with-nextjs-and-clerk-auth/src/themes/material.ts

```diff
@@ -0,0 +1,27 @@
+import type { ThemeOptions } from '@mui/material/styles';
+import type { Components } from '@mui/material/styles/components';
+import type { Theme } from '@mui/material/styles/createTheme';
+
+const components: Components<Omit<Theme, 'components'>> = {
+  MuiList: {
+    styleOverrides: {
+      root: {
+        '&.navigation-list': {
+          '.MuiCollapse-root .MuiListItemButton-root': {
+            paddingLeft: 28,
+          },
```

Comment:
I'd rather you add styling directly in the component, rather than at the theme level. The MUI theme should be exactly the same as in `packages/sdk-themes`:

```ts
import type { ThemeOptions } from '@mui/material/styles/createTheme.js';
export const materialLight: ThemeOptions = { palette: { mode: 'light' } };
export const materialDark: ThemeOptions = { palette: { mode: 'dark' } };
```